### ISSUE TITLE
golem.de

### ISSUE THREAD

**Issue by bafa1012** [reek/anti-adblock-killer#2062]
[golem.de](http://www.golem.de/news/apple-neues-iphone-7-plus-kommt-mit-dual-kamera-1609-123140.html)
http://imgur.com/a/W5og2

Google Chrome
ublock origin
Tampermonkey with anti-adblock-killer script

filters:
 - EasyList
 - CHN: EasyList China (中文)
 - CHN: CJX's EasyList Lite (main focus on Chinese sites)
 - CHN: CJX's Annoyance List
 - DEU: EasyList Germany
 - EasyPrivacy
 - Fanboy’s Annoyance List
 - Fanboy’s Social Blocking List
 - hpHosts’ Ad and tracking servers
 - Malvertising filter list by Disconnect
 - Malware filter list by Disconnect
 - Basic tracking list by Disconnect
 - Adblock Warning Removal List
 - Malware domains (long-lived)
 - Malware domains
 - Peter Lowe’s Ad server list
 - Anti-Adblock Killer | Reek
 - Dan Pollock’s hosts file
 - Spam404
 - MVPS HOSTS
 - Fanboy’s Enhanced Tracking List
 - Anti-ThirdpartySocial (see warning inside list)
 - Fanboy+Easylist-Merged Ultimate List
 - Malware Domain List

**Comment by yoshimo** [comment]
@bafa1012 , you manuallyn blocked ecruos.golem.de to hide the big abo banners, didn't you?

**Comment by EpicLPer** [comment]
Yes, please, it's annoying as hell and when they're doing it like this I'll definitely NOT pay for a abo...
Besides that, their articles aren't that "super duper great" most of the times anyways.

**Comment by EpicLPer** [comment]
More info about this:

Once you block "ecruos.golem.de" Golem will check if it's still reachable, if not it'll trigger a script (not sure which one yet) and display a blurred text over the article.

The blurred text hooks into the "article" tag and adds a style with `filter: blur(8px) grayscale(80%) opacity(40%); pointer-events: none; position: absolute; top: 0px;` into the article tag.
It also adds a div with class "formatted" and a div with `class="dh1 head5">ADBLOCKER ERKANNT`, plus some extra content.


Hope this helps a bit.

**Comment by yoshimo** [comment]
even more:

`golem.de##script:contains(_sp_)
forum.golem.de##script:contains(_sp_)
video.golem.de##script:contains(_sp_)`
does work rather well for now in Ublock Origin

**Comment by EpicLPer** [comment]
@yoshimo Nope, doesn't work for me for some reason, after blocking ecruos.golem.de the adblock blurred text comes up again.

**Comment by Strubbl** [comment]
do not block ecruos.golem.de. use @yoshimo solution (and add a newline for each domain)
works for me:
```
golem.de##script:contains(_sp_)
forum.golem.de##script:contains(_sp_)
video.golem.de##script:contains(_sp_)
```

**Comment by EpicLPer** [comment]
Nope, still not working :(

**Comment by Strubbl** [comment]
Maybe some other rule of yours interferes with the above mentioned golem rules. I have tested it to be working with Linux/Firefox and Windows7/Firefox on different machines and it works as expected.
